remote/learners/jaswir.md

```diff
@@ -0,0 +1,2 @@
+Hello K8S!
+Jaswir Raghoe here from the Netherlands. 
```

Comment:
Please remove changes to this file

Reply:
I  had updated the commit message with changes